### ISSUE TITLE
Question: Is there a way I can extend a running emitter's lifespan?

### ISSUE THREAD

**Issue by defmech** [creativelifeform/three-nebula#42]
Use case: I have an emitter that runs for 10 seconds. I'd like to extend the life of it by another 10 seconds after it's already started. Not reset it but extend its lifespan after it's started. I've tried resetting age and totalEmitTimes to 0 but it doesn't seem to affect run time.

**Comment by rohan-deshpande** [comment]
Are you saying you want to be able to emit again after the emitter has reached the end of its life? This is currently not supported but will be soon, see #31

**Comment by defmech** [comment]
No, In my use case I have an animation that's triggered by an action. Particles fall from the sky for 10 seconds. If the user triggers the action again, while the particles are still falling, I'd like to add another 10 seconds to the life.

Example.

User triggers falling particles that last 10s.
5s later they trigger them again.
I'd like to update emitter so that the total runtime is 15s.

Does that make sense?

**Comment by rohan-deshpande** [comment]
Ok yep, makes sense. Have you tried updating the emitter life directly at that point? `totalEmitTimes` might not be what you want but rather just simply writing a function that adds 10 seconds (or whatever period of time you want) to `life` when the event fires?

**Comment by rohan-deshpande** [comment]
Closing due to inactivity
